### PR TITLE
RR 2.0 example snippet

### PR DESCRIPTION
**Pull request by ev1stensberg**
Added RR 2.0 example

### CODE REVIEW

File: docs/api.md [hunk 1]
```diff
@@ -49,6 +49,22 @@ ReactDOM.render(
...
+      <Route path="/" component={App}>
+        <Route path="foo" component={Foo}/>
+        <Route path="bar" component={Bar}/>
+      </Route>
+    </Router>
+  </Provider>,
+  document.getElementById('root')
+)
+```
+
```
Comment (ev1stensberg) [inline docs/api.md]: One line too much?

File: docs/api.md [hunk 2]
```diff
@@ -49,6 +49,22 @@ ReactDOM.render(
...
+  <Provider store={store}>
+    <Router history={history}>
+      <Route path="/" component={App}>
+        <Route path="foo" component={Foo}/>
+        <Route path="bar" component={Bar}/>
+      </Route>
+    </Router>
+  </Provider>,
+  document.getElementById('root')
+)
```
Comment (ev1stensberg) [inline docs/api.md]: One line too much+

**Comment by ev1stensberg** [comment]
~ This snippet was taken from somewhere else in the Repo, but I think it is convenient to have it here as well.

**Comment by ev1stensberg** [comment]
It's also the same, but people may think it is different from 1.0. Alternatively we can change `React Router 1.0` to `2.0`

**Comment by timdorr** [comment]
I'll just rename this to remove the version part.

**Pull request merged**